### ISSUE TITLE
permission denied for relation team_rating

### ISSUE THREAD

**Issue by Chipitos** [odota/core#1642]
Hello. 
I have request that previously worked, but now getting error
https://api.opendota.com/api/explorer?sql=SELECT teams.team_id, teams.name, teams.tag FROM teams LEFT JOIN team_rating ON teams.team_id=team_rating.team_id WHERE team_rating.last_match_time > 1514764800

Error: {"err":"error: permission denied for relation team_rating}

**Comment by Chipitos** [comment]
I thought i need to get api key, but with api key i still get same result

**Comment by Chipitos** [comment]
@howardchung Yeah, it works, but i need only teams, that played at least one match after 1 January,2018 and i can get it only with explorer.

**Comment by howardchung** [comment]
perhaps you could do the query without team rating and then add the team rating to the teams after?

alternatively i think we can allow access to the team_rating table, but it will need to wait for the next database update.

**Comment by Chipitos** [comment]
@howardchung Sure, it i can wait until next update. Thank you!

**Comment by howardchung** [comment]
The match_gcdata table is not intended for public consumption, you can use the /replays endpoint to fetch that data.

Don't have a timeframe for when the database will be updated, it usually depends on when we need to make a schema change to add new data, etc.

**Comment by Chipitos** [comment]
@howardchung So, how i can get series id from explorer for grouping matches score? 
For now, i need to get results between two teams by id and show score for series.

**Comment by howardchung** [comment]
series ID is supposed to be moved to the matches table.  I haven't gotten around to doing it yet unfortunately:

https://github.com/odota/core/issues/1604

**Comment by Chipitos** [comment]
Hi @howardchung . Any update by #1604 issue?
